vcpkg/contributing/maintainer-guide.md

```diff
@@ -50,16 +52,26 @@ then obviously beneficial changes like fixing typos are appreciated!
 
 ### Check names against other repositories
 
-A good service to check many at once is [Repology](https://repology.org/).
-If the library you are adding could be confused with another one,
-consider renaming to make it clear. We prefer when names are longer and/or
-unlikely to conflict with any future use of the same name. If the port refers
-to a library on GitHub, a good practice is to prefix the name with the organization
-if there is any chance of confusion.
+Port names should, as much as possible, be unambiguous about the package the port
+installs. When a specific name is strongly associated with a single project the
+port can simply take the name of the package. For example, `openssl`, `tensorflow`,
+and `zlib`.
 
-Put another way, the reason for this is to ensure that `vcpkg install Xxx`
-gives the user looking for `Xxx` what they were expecting and not be
-surprised by getting something different.
+Some times a package may use a common word as the package's name, especially if

```

Comment:
I think this paragraph could be reworded to more clearly state the intent in the first sentence.

For example: "Projects with short names or common words may require disambiguation. ...."